src/components/prescriptions/PrescriptionDetails.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div
+        v-bind:key="props.id"
+        class="detailsPres">
+        <h3 class="detailsPres__header">Szczegóły recepty</h3>
+        <form class="detailsPres__form">
+            <div class="inputBox">
+                <h5 class="detailsPres__title">
+                    <label> Lekarz: </label>
+                </h5>
+                <p class="detailsPres__data">{{ names }}</p>
+            </div>
+
+            <div class="inputBox">
+                <h5 class="detailsPres__title">
+                    <label> Nazwa leku: </label>
+                </h5>
+                <p class="detailsPres__data">{{ medicines }}</p>
+            </div>
+
+            <div class="inputBox">
+                <h5 class="detailsPres__title">
+                    <label> Częstotliwość przyjmowania: </label>
+                </h5>
+                <p class="detailsPres__data">{{ frequencys }}</p>
+            </div>
+
+            <div class="inputBox">
+                <h5 class="detailsPres__title">
+                    <label>Data wystawienia recepty:</label>
```

Comment:
good label changes but there is no neeed to add <label> inside. I meant label about value in h5 not adding additional markup ;)